apps/registration/src/hooks/__tests__/useCreatePatient.test.tsx

```
@@ -125,256 +128,150 @@ describe('useCreatePatient', () => {
-      const formDataWithoutBirthTime = {
+    it('should treat photo upload failure as non-fatal — patient create still succeeds', async () => {
+      mockCreateFhirPatient.mockResolvedValue(mockFhirPatientResponse);
+      mockUploadPatientPhoto.mockRejectedValue(new Error('Upload failed'));
+
+      const formDataWithImage = {
         ...mockFormData,
-        profile: {
-          ...mockFormData.profile,
-          birthTime: '',
-        },
+        profile: { ...mockFormData.profile, image: 'data:image/jpeg;base64,abc123' },
```

ESLint

[error] Replace `·...mockFormData.profile,·image:·'data:image/jpeg;base64,abc123'` with `⏎··········...mockFormData.profile,⏎··········image:·'data:image/jpeg;base64,abc123',⏎·······`

(prettier/prettier)